ray-operator/controllers/ray/common/ingress.go

```diff
@@ -42,10 +42,10 @@ func BuildIngressForHeadService(ctx context.Context, cluster rayv1.RayCluster) (
 
 	var paths []networkingv1.HTTPIngressPath
 	pathType := networkingv1.PathTypeExact
-	servicePorts := getServicePorts(cluster)
+	serviceMapPortInfo := getServicePorts(cluster)
```

Comment:
`serviceMapPortInfo` sounds a little wired. How about we keep it as `servicePorts`?

Reply:
Adjusted